### PR TITLE
Fix golint failures

### PR DESCRIPTION
**Pull request by xichengliudui**
**What type of PR is this?**
 /kind cleanup

**What this PR does / why we need it**:
Fix golint failures

**Which issue(s) this PR fixes**:

Ref #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by neolit123 — APPROVED** [review]
/kind cleanup
/priority backlog
/lgtm

File: pkg/kubelet/util/sliceutils/sliceutils.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ import (
 	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
 )
 
+// StringInSlice traversal and determine return bool
```
Comment (BenTheElder) [inline pkg/kubelet/util/sliceutils/sliceutils.go]: perhaps instead:
```suggestion
// StringInSlice returns true if s is in list
```

File: pkg/kubelet/util/queue/work_queue.go [hunk 2]
```diff
@@ -41,6 +41,7 @@ type basicWorkQueue struct {
 
 var _ WorkQueue = &basicWorkQueue{}
 
+// NewBasicWorkQueue instantiate the queue and return WorkQueue
```
Comment (BenTheElder) [inline pkg/kubelet/util/queue/work_queue.go]: perhaps
```suggestion
// NewBasicWorkQueue returns a new basic WorkQueue with the provided clock
```

Comment (xichengliudui) [reply pkg/kubelet/util/queue/work_queue.go]: It has been modified, thank you

**Review by BenTheElder — APPROVED** [review]
/lgtm
Thanks

**Review by BenTheElder — APPROVED** [review]
/lgtm
Thanks

**Pull request merged**